src/lib/Logger/index.ts

```diff
@@ -37,6 +37,13 @@ export enum ErrorLocation {
   IPFSSERVICE_UPLOADDIRECTORY = '[IPFS.uploadDirectory]',
   IPFSSERVICE_UNPINFILE = '[IPFS.unPinFile]',
 
+  // IPFS SERVER CLIENT
+  IPFS_SERVER_CONSTRUCTOR = '[IPFS.constructor]',
+  IPFS_SERVER_UPLOADFILE = '[IPFS.uploadFile]',
+  IPFS_SERVER_UPLOADOBJECT = '[IPFS.uploadObject]',
```

Comment:
where is the IPFS_SERVER_UPLOADOBJECT  used?

Reply:
no where , we don't need it , i'll remove it now

Reply:
🙏